src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:  
+          <input 
+            type="text"
+            class="surname" 
+            name="surname" 
+            required
+          >
+        </label>
+
+        <div class="one">  
+          <label>
+            Name:  
+            <input 
+              type="text"
+              class="name"
+              name="name" 
+              required
+            >      
+          </label>
+        </div>
```

Comment:
Classes should have more meaningful names. And do you really need additional `div`s everywhere? 

```suggestion
        <label class="form-field">
          Name:  
          <input 
            type="text"
            class="name"
            name="name" 
            required
          >      
        </label>
```

Reply:
Without divs, inputs get in one row.

Reply:
Lora Levitska
:mate:  [4 days ago](https://mate-parttime.slack.com/archives/C03UQS59NRY/p1674904800526529?thread_ts=1674901535.711959&cid=C03UQS59NRY)
привіт)
Додай клас а не навішуй стилі для тегу.
choice1 - це вказівка атрибуту для якого id ти застосовуєш, а не клас

Reply:
labels is inline elements. They don't make blocks